ignite/cmd/chain.go

```diff
@@ -97,10 +102,67 @@ chain.
 	return c
 }
 
-func configMigrationPreRunHandler(cmd *cobra.Command, _ []string) (err error) {
+func preRunHandler(cmd *cobra.Command, _ []string) error {
 	session := cliui.New()
 	defer session.End()
 
+	if err := configMigrationPreRunHandler(cmd, session); err != nil {
+		return err
+	}
+	return toolsMigrationPreRunHandler(cmd, session)
+}
+
+func toolsMigrationPreRunHandler(cmd *cobra.Command, session *cliui.Session) (err error) {
+	session.StartSpinner("Checking missing tools...")
+
+	appPath := flagGetPath(cmd)
+	toolsFilename := filepath.Join(appPath, toolsFile)
+	f, _, err := xast.ParseFile(toolsFilename)
+	if err != nil {
+		return err
+	}
+
+	var (
+		missing = cosmosgen.MissingTools(f)
+		unused  = cosmosgen.UnusedTools(f)
+	)
```

Comment:
Maybe
```suggestion
	missing := cosmosgen.MissingTools(f)
	unused  := cosmosgen.UnusedTools(f)
```

Reply:
Do you have any particular reason for it? Grouping some variables can be better for readability, but we can use the same if we have any code style to follow